external/fv3gfs-fortran.requirements.in

```diff
@@ -4,7 +4,7 @@ xarray>=0.15.1
 netCDF4>=1.4.2
```

Comment:
the fv3gfs-wrapper submodule was moved into fv3gfs-fortran in #1805 and presumably the dependencies remain the same, but it's easier to follow if this file refers to the current submodule